benchmark_runner/main/main.py

```diff
@@ -64,6 +122,20 @@ def main():
                                                              pin_node='pin_node_benchmark_operator')
         benchmark_operator_workload.run_workload(workload=workload)
 
+    # azure_cluster_start_stop
+    if azure_cluster_stop or azure_cluster_start:
+        azure_cluster_start_stop()
+    # install_ocp_version
+    elif install_ocp_version:
+        install_ocp()
+    # install_ocp_resource
+    elif install_ocp_resources:
+        install_resources()
+    elif ci_status == 'pass' or ci_status == 'failed':
+        update_ci_status()
+    else:
+        run_workload()
```

Comment:
Check here for missing workload?

Reply:
That's at top level, so it's going to throw a stack trace.  Since this is a user error, perhaps at least catch all exceptions and log them, so it's a bit cleaner?  Not a huge issue, but think about it.

Reply:
All should run automatically with right parameters.
The user will only run a workload and he has a clear instructions for it in readme.